src/topology/locally_constant/algebra.lean

```diff
@@ -72,6 +72,44 @@ instance [mul_zero_class Y] : mul_zero_class (locally_constant X Y) :=
 instance [mul_zero_one_class Y] : mul_zero_one_class (locally_constant X Y) :=
 { .. locally_constant.mul_zero_class, .. locally_constant.mul_one_class }
 
+section char_fn
+
+variables (Y) [mul_zero_one_class Y] {U V : set X}
+
+/-- Characteristic functions are locally constant functions taking `x : X` to `1` if `x ∈ U`,
+  where `U` is a clopen set, and `0` otherwise. -/
+noncomputable def char_fn (hU : is_clopen U) : locally_constant X Y := indicator 1 hU
+
+lemma char_fn_one [nontrivial Y] (x : X) (hU : is_clopen U) :
```

Comment:
I think this lemma should instead be proved as a fact about `set.indicator U 1` since it doesn't really matter that `U` is clopen.

Likewise the two lemmas below.

Reply:
Done, thanks!